### PR TITLE
Change cmake from string to array<string>

### PR DESCRIPTION
**Pull request by ladisgin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # What is UTBotCpp?
+
+[![Build UTBot and run unit tests](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/build-utbot.yml/badge.svg)](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/build-utbot.yml)
+[![Publish UTBot as an archive](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/publish-utbot.yml/badge.svg)](https://github.com/UnitTestBot/UTBotCpp/actions/workflows/publish-utbot.yml)
+
```
Comment (operasfantom) [inline README.md:5]: Is it related to the issue? If not, please move to a separate commit or PR

**Review by operasfantom — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ladisgin) [reply README.md]: Delete from commit

**Review by operasfantom — APPROVED** [review]
